docs/managed_vms/endpoints/README.md

```diff
@@ -1,8 +1,27 @@
 # Google Cloud Endpoints sample
 
+WARNING: This sample does not work with Go 1.7 yet. Please use Go 1.6.
+
+## Updating the project ID
+
+The project ID in the sample must be updated to match your project ID.
+
+Edit the `swagger.yaml` file and replace `YOUR-PROJECT-ID` with your project ID.
+
 ## Deploying the backend
 
-Follow the [instructions][deploy] in the docs.
+To deploy a Go app, use the [aedeploy
+tool](https://godoc.org/google.golang.org/appengine/cmd/aedeploy), which will
+correctly assemble your app's dependencies in the same way that the go tool
+does. From this sample's directory, install aedeploy by running:
+
+    export GOPATH=`pwd`
+    go get -u -d github.com/GoogleCloudPlatform/golang-samples/docs/managed_vms/endpoints/...
+    go get google.golang.org/appengine/cmd/aedeploy
+
+Deploy your app by appending the `gcloud` command after `aedeploy`:
+
+    bin/aedeploy gcloud beta app deploy
```

Comment:
This was required because GOPATH was set to the current directory and GOTPATH/bin is not in the PATH.